src/compiler/utilities.ts

```diff
@@ -3236,6 +3240,14 @@ namespace ts {
         return isSingleOrDoubleQuote(name.charCodeAt(0));
     }
 
+    export function stripQuotesOrBacktics(name: string) {

```

Comment:
Had to create duplicate of `stripQuotes` because that one is used in 7 other places. Quick look at those places and passing test after update to `stripQuotes` suggest that there should be no harm in function update.
Should I update `stripQuotes` and its JSDoc instead of creating separate function?

Reply:
I think it should be ok to update `stripQuotes` 👍

Reply:
Good, will update this one today.

Reply:
Fixed